### PR TITLE
Improve the phrasing of the page "strong consistency"

### PR DESCRIPTION
**Pull request by knz**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/1002)

### CODE REVIEW

**Comment by jseldess** [comment]
I'll wait for @bdarnell's feedback, but this looks great to me. Would you mind updating https://www.cockroachlabs.com/docs/frequently-asked-questions.html#how-is-cockroachdb-strongly-consistent as well, @knz ?

File: strong-consistency.md [hunk 1]
```diff
@@ -4,15 +4,44 @@ summary: CockroachDB implements consistent replication via majority consensus be
...
+  This ensures that clients always observe serializable consistency
+  with regards to other concurrent transactions.
+
 <img src="images/2strong-consistency.png" alt="Strong consistency in CockroachDB" style="width: 400px" />
 
 ## See Also
 
 - [Consensus, Made Thrive](https://www.cockroachlabs.com/blog/consensus-made-thrive/)
 - [Trust, But Verify: How CockroachDB Checks Replication](https://www.cockroachlabs.com/blog/trust-but-verify-cockroachdb-checks-replication/)
+- [Serializable, Lockless, Distributed: Isolation in CockroachDB](https://www.cockroachlabs.com/blog/serializable-lockless-distributed-isolation-cockroachdb/)
```
Comment (bdarnell) [inline strong-consistency.md]: I'd move this one up to the top of the list; I think it's the most important thing to understand about our consistency model.

**Comment by knz** [comment]
> I'd move this one up to the top of the list; I think it's the most
> important thing to understand about our consistency model.
> 

done

**Comment by knz** [comment]
I have updated the FAQ as requested by @jseldess. PTAL

**Comment by knz** [comment]
Feel free to merge then =)

**Comment by jseldess** [comment]
@knz, was pulled away by kids, etc. You should feel free to merge in the future as well.

**Pull request merged**